Makefile

```diff
@@ -11,36 +17,60 @@ generate:
 # If you are working on changes to protobuf interfaces, run this Makefile
 # target to be sure to regenerate all of the protobuf stubs using the expected
 # versions of protoc and the protoc Go plugins.
+.PHONY: protobuf
 protobuf:
 	go run ./tools/protobuf-compile .
 
+.PHONY: fmtcheck
 fmtcheck:
 	"$(CURDIR)/scripts/gofmtcheck.sh"
 
+.PHONY: importscheck
 importscheck:
 	"$(CURDIR)/scripts/goimportscheck.sh"
 
+.PHONY: staticcheck
 staticcheck:
 	"$(CURDIR)/scripts/staticcheck.sh"
 
+.PHONY: exhaustive
 exhaustive:
 	"$(CURDIR)/scripts/exhaustive.sh"
 
 # Run this if working on the website locally to run in watch mode.
+.PHONY: website
 website:
 	$(MAKE) -C website website
 
 # Use this if you have run `website/build-local` to use the locally built image.
+.PHONY: website/local
 website/local:
 	$(MAKE) -C website website/local
 
 # Run this to generate a new local Docker image.
+.PHONY: website/build-local
 website/build-local:
 	$(MAKE) -C website website/build-local
 
+# Run license check
+.PHONY: license-check
+license-check:
+	go mod vendor
+	licensei check
+	licensei header
```

Comment:
We may want to add header rules: https://github.com/goph/licensei/tree/v0.9.0#configuration-example